Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM node:lts-alpine
+
+RUN \
+  apk update && \
```

Comment:
This is not needed when using `--no-cache` since package list is downloaded every time.